File: 코딩테스트 연습 - 2016년 _ 프로그래머스.java
class Solution {
    public String solution(int a, int b) {
        String answer = "";
        int days = 0;
        String[] results = {"THU", "FRI","SAT", "SUN","MON","TUE","WED"};
        int[] months = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        for(int i = 0; i < a-1; i++) {
            days += months[i];
        }
        days += b;
        
        answer = results[days%7];
        
        return answer;
    }
}

File: python/코딩테스트 연습 - 2016년 _ 프로그래머스.py
def solution(a, b):
    answer = ''
    days = ["THU", "FRI","SAT", "SUN","MON","TUE","WED"]
    months = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    d = sum([months[i] for i in range(a-1)])
    d += b
    return days[d % 7]